pydash/arrays.py

```diff
@@ -1443,6 +1487,58 @@ def union(*arrays):
     return uniq(flatten(arrays))
 
 
+def union_by(*arrays, **kargs):
```

Comment:
Look at intersection_by(), for example, and just duplicate the handling of others and kargs. Basically the callback should be passable as either a positional arg or a keyword arg.